lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -220,6 +225,8 @@ def test_get(self):
         data = response.data
         courses = data['results']
 
+        print courses
```

Comment:
print statement?
(Sorry for lurking in here.. was looking for something else.. :)